server/src/games/BGS.Games.StoneAge/StoneAgeScoringModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace BGS.Games.StoneAge;
+
+public record StoneAgeScoringModel
+{
+    public short CurrentPoint { get; init; }
+
+    public short CountOfResources { get; init; }
+
+    public byte NumberOfFoodProduction { get; init; }
+
+    public byte NumberOfFarmers { get; init; }
+
+    public byte NumberOfTools { get; init; }
+
+    public byte NumberOfToolMakers { get; init; }
+
+    public byte NumberOfBuildings { get; init; }
+
+    public byte NumberOfHutBuilders { get; init; }
+
+    public byte NumberOfShamens { get; init; }
```

Comment:
Sham**a**ns =)

Reply:
yes, this misspell is in the rules of the game itself :(